src/scripts/main.js

```diff
@@ -5,4 +5,17 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  table.insertAdjacentHTML('beforeend', `
+  <tr>
+    <th>${person.name}</th>
+    <th>${person.sex}</th>
+    <th>${person.born}</th>
+    <th>${person.died}</th>
+    <th>${person.died - person.born}</th>
+    <th>${Math.ceil(person.died / 100)}</th>
```

Comment:
you need replace 'th' for 'td', because 'th' means table header)

Reply:
My bad😅